tests/test_cli/test_cfg_subcmd.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import os
+import shutil
+
+from test_cli import capture
+this_folder = os.path.dirname(__file__)
+
+def test_cfg_help(python_version):
+    """Test if Manim successfully adds configparsers when a subcommand is invoked."""
+    command = f"cd {this_folder} && {python_version} -m manim cfg --help"
+    out, err, exitcode = capture(command, use_shell=True)
+    assert exitcode == 0, f"The cfg subcommand is not working as intended."
```

Comment:
**Issue :** Add err somewhere in the message. It will be way way more faster to debug, believe me.
```suggestion
    assert exitcode == 0, f"The cfg subcommand help is not working as intended.\nError : {err}"
```